ironfish/src/mining/manager.ts

```diff
@@ -104,8 +104,12 @@ export class MiningManager {
         nullifiers.add(spend.nullifier)
       }
 
-      // Stop adding transactions when the addition would cause the block to exceed the max size
       const transactionSize = getTransactionSize(transaction.serialize())
+      if (transactionSize > this.chain.consensus.MAX_BLOCK_SIZE_BYTES) {
+        continue
+      }
```

Comment:
Should our mem pool just reject these in its acceptFunction?

Reply:
Agree, and the practical limit will be something like MAX_BLOCK_SIZE - block header - miner fee (though it'll fail the below check anyway)

Reply:
Updated to reject these transaction in acceptTransaction and to use (MAX_BLOCK_SIZE  - block header size - miner's fee transaction size - transaction length size) limit. Right now I'm using 562 and 2 bytes directly as the values for the miner's fee transaction size and transaction length size in `manager.ts` and `memPool.ts`, do you guys think it would be better if we had these as exported consts somewhere?

Reply:
Yeah, that's a good idea. Now we'll be doing (blockheader + 562 + 2) in several places as well, so I think it'd be better to have that calculation centralized somewhere.

Another thing that came to mind is that miner's fee transactions aren't guaranteed to be a fixed size. I don't think this needs to change since we don't currently support generating miner transactions with multiple outputs, but just something to keep in mind for the future.